### DISCUSSION TITLE
Add more feature for resign command

### DISCUSSION THREAD

**Discussion by nghiaiosdev** [fastlane/fastlane#17872] [Ideas]
Currently. I see resign command working well for resign api. But I also see that, resign command just makes codesign for the main executable and dylib and framework located inside Frameworks folder. With dylib outside Framework folder, resign command dont working well. I open this discussion to request add codesign for dylib and frameworks located anywhere in Payload folder. Thank in advance.

**Comment by rogerluan** [comment]
Hey @eitguide 👋 

Please create a [new issue](https://github.com/fastlane/fastlane/issues/new?assignees=&labels=&template=BUG_REPORT.md) filling up all the environment and command output information so we can better assist you 🙏
